### PR TITLE
addpkg(main/far2l): FAR Manager v2 - Linux port

### PR DESCRIPTION
**Pull request by spvkgn**
[GitHub](https://github.com/elfmz/far2l)
[Repology](https://repology.org/project/far2l/versions)

### CODE REVIEW

**Comment by 2096779623** [comment]
I suggest you to use [this patch](https://github.com/termux/termux-packages/blob/3c291eb3b2e3859df8f4563ef47280e1568ca497/packages/7zip/disable_hardware_acceleration_support_on_arm.patch#L39) to fix arm build.

**Review by truboxl — CHANGES_REQUESTED** [review]
This package should be moved to `x11-packages`

**Comment by truboxl** [comment]
I will help rebasing branch since CI got polluted

File: x11-packages/far2l/build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/elfmz/far2l
+TERMUX_PKG_DESCRIPTION="FAR Manager v2 - Linux port"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@spvkgn"
+TERMUX_PKG_VERSION=2.5.1
+TERMUX_PKG_REVISION=1
```
Comment (truboxl) [inline x11-packages/far2l/build.sh:6]:
```suggestion
```
Not needed

File: x11-packages/far2l/build.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+TERMUX_PKG_SRCURL=https://github.com/elfmz/far2l/archive/v_${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6a4456c84be0971dd0e929903fc4b16891881f0d8c2f6483c12138c51cd6387d
+TERMUX_PKG_DEPENDS="libc++, libarchive, libuchardet, pcre"
```
Comment (truboxl) [inline x11-packages/far2l/build.sh:9]:
```suggestion
TERMUX_PKG_DEPENDS="libarchive, libc++, libuchardet, pcre"
```

File: x11-packages/far2l/build.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+TERMUX_PKG_SUGGESTS="chafa, exiftool, htop, timg"
+TERMUX_PKG_RM_AFTER_INSTALL="share/icons share/applications"
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DCMAKE_BUILD_TYPE=Release
```
Comment (truboxl) [inline x11-packages/far2l/build.sh:15]:
```suggestion
```
Not needed

File: x11-packages/far2l/far2l-plugins-netrocks.subpackage.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+TERMUX_SUBPKG_DESCRIPTION="FAR2L NetRocks plugin (SFTP/SCP/FTP/SMB/NFS/WebDAV)"
+TERMUX_SUBPKG_DEPENDS="libneon, libnfs, libssh, openssl, samba"
```
Comment (truboxl) [inline x11-packages/far2l/far2l-plugins-netrocks.subpackage.sh:2]:
```suggestion
TERMUX_SUBPKG_DEPENDS="libc++, libneon, libnfs, libssh, openssl, samba"
```

File: x11-packages/far2l/far2l-ttyxi.subpackage.sh [hunk 5]
```diff
@@ -0,0 +1,3 @@
+TERMUX_SUBPKG_DESCRIPTION="X11/Xi extension to provide best UX for TTY backend"
+TERMUX_SUBPKG_DEPENDS="libxi"
```
Comment (truboxl) [inline x11-packages/far2l/far2l-ttyxi.subpackage.sh:2]:
```suggestion
TERMUX_SUBPKG_DEPENDS="libc++, libx11, libxi"
```

File: x11-packages/far2l/build.sh [hunk 6]
```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/elfmz/far2l
+TERMUX_PKG_DESCRIPTION="FAR Manager v2 - Linux port"
```
Comment (truboxl) [inline x11-packages/far2l/build.sh:2]:
```suggestion
TERMUX_PKG_DESCRIPTION="FAR Manager v2"
```
I think a general description is enough

**Review by truboxl — CHANGES_REQUESTED** [review]

**Review by truboxl — APPROVED** [review]
So far LGTM thanks! Hopefully you can upstream your patches.

I will help squash and merge if others have no more reviews.

**Pull request merged**